app/qml/inputs/MMTextAreaEditor.qml

```diff
@@ -0,0 +1,51 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Controls.Basic
+import QtQuick.Layouts
+import Qt5Compat.GraphicalEffects
+import "../components"
+
+MMAbstractEditor {
+  id: root
+
+  property var parentValue: parent.value ?? ""
+  property bool parentValueIsNull: parent.valueIsNull ?? false
+  property bool isReadOnly: parent.readOnly ?? false
+
+  property alias placeholderText: textArea.placeholderText
+  property alias text: textArea.text
+
+  signal editorValueChanged( var newValue, var isNull )
+
+  hasFocus: textArea.activeFocus
+  contentItemHeight: {
+    const minHeight = 85 * __dp // min height for 3 rows
```

Comment:
We'll need to test out if this constant means 3 rows on all platforms. I'd prefer calculating it from font size, but let's see :)